docs/docs/cmd/spo/folder/folder-retentionlabel-ensure.md

```diff
@@ -0,0 +1,47 @@
+# spo folder retentionlabel ensure
+
+Apply a retention label to a folder
+
+## Usage
+
+```sh
+m365 spo folder retentionlabel ensure [options]
+```
+
+## Options
+
+`-u, --webUrl <webUrl>`
+: URL of the site where the retentionlabel from a file to apply is located
+
+`--folderUrl [folderUrl]`
+: The server-relative URL of the folder that should be labelled. Specify either `folderUrl` or `folderId` but not both.
```

Comment:
This should be a site- or server relative url. We've got a util that you can use to convert that easily: `urlUtil.getServerRelativePath`